test/helpers/sign.js

```diff
@@ -1,7 +1,4 @@
-const REAL_SIGNATURE_SIZE = 2 * 65; // 65 bytes in hexadecimal string legnth
-const PADDED_SIGNATURE_SIZE = 2 * 96; // 96 bytes in hexadecimal string length
-
-const DUMMY_SIGNATURE = `0x${web3.utils.padLeft('', REAL_SIGNATURE_SIZE)}`;
+let web3;
```

Comment:
Why can't we require test-environment here?

Reply:
Hm, we could. I think I started the migration before `web3` was in the test-environment API, and didn't notice after.